packit/config/common_package_config.py

```diff
@@ -205,14 +206,20 @@ def __init__(
         # this is generated by us
         self.downstream_package_name: Optional[str] = downstream_package_name
         self._downstream_project_url: str = downstream_project_url
-        self.dist_git_base_url: str = dist_git_base_url or getenv(
-            "DISTGIT_URL",
-            PROD_DISTGIT_URL,
+
+        # Set up the dist-git instance parameters
+        # XXX: Try to think about something more reasonable… UGH
+        self.dist_git_base_url: str = (
+            dist_git_base_url
+            or (pkg_tool and DISTGIT_INSTANCES[pkg_tool].url)
+            or getenv("DISTGIT_URL", DISTGIT_INSTANCES["fedpkg"].url)
```

Comment:
I'd probably have a look at hardly, if it makes sense to default to the packaging tool from the user config, rather than the environment variable.